securedrop/tests/test_journalist_api.py

```diff
@@ -26,7 +26,7 @@
 
 def assert_valid_timestamp(timestamp: str) -> None:
     """verify the timestamp is encoded in the format we want"""
-    dt_format = "%Y-%m-%dT%H:%M:%S.%fZ"
+    dt_format = "%Y-%m-%dT%H:%M:%S.%f+00:00"
```

Comment:
Can we use `datetime.fromisoformat()` to match securedrop-sdk? Looking at https://github.com/freedomofpress/securedrop-sdk/blob/31ea11e3f9d8c243c0fbe2a19bbdd4f75bd32bd0/sdclientapi/timestamps.py

Reply:
Yes, we can! I left this one as-is because I like having distinct implementations for the tests and the code under test. I found that using the format string made particularly clear the expectation in the test file. But all that being said, I'd say as a `securedrop` repo maintainer it's your call, and I don't personally have strong feelings about it.

Reply:
Maybe both then? I would like to make sure we're at least testing with what the sdk is actually using, which is now `fromisoformat()`.